docs/book/v3/reference/migration.md

```diff
@@ -18,6 +18,14 @@ previously might have worked will now raise a `TypeError`.
 The following signatures changed that could affect class extension and/or
 consumers.
 
+### Interface suffix
+
+As of
+[laminas-coding-standard](https://github.com/laminas/laminas-coding-standard)
+version `2.0`, interface names must have an `Interface` suffix. The `Server`
```

Comment:
I wouldn't mention that this is a change dictated by laminas-coding-standard; I'd mention it's being done for consistency with the rest of the project, where we have standardized on the `Interface` suffix.

Reply:
Done.